CMakeLists.txt

```diff
@@ -187,3 +187,57 @@ if(WIN_BUILD)
     set(ENV{PKG_CONFIG_PATH} "${OUTPUTDIR}")
 	message("PKG_CONFIG_PATH: " $ENV{PKG_CONFIG_PATH})
 endif()
+
+# Windows installer build
+if(WIN_BUILD AND RELEASE)
```

Comment:
Please update README.md file, add info how to build your own NSIS installer for Windows using cmake (may be added below the information about building Windows binaries)